toolboxes/core/cpu/hoNDObjectArray.h

```diff
@@ -1,6 +1,6 @@
 /** \file   hoNDObjectArray.h
-\brief  CPU-based N-dimensional array for object pointers
-if delete_data_on_destruct == true, the object will be released; otherwise, only the object array memory is released
+\brief  N-dimensional array for objects
```

Comment:
This file should be removed completely, I think. There will be a few Gadgets, etc. to update after that, but we should complete the removal of this typename throughout.

Reply:
Agreed, but I suggested we put the removal of this file to another PR. So it will only change there.

Reply:
Not sure I get why. It would be a search and replace to update that. Seems like there is more risk of it never getting done.

Reply:
ok, removed this file and all its reference.